src/llrt-function.ts

```diff
@@ -130,7 +137,7 @@ export class LlrtFunction extends NodejsFunction {
               rm -rf llrt_temp.zip
              fi`,
             `cp ${posix.join(i, cacheDir, 'bootstrap')} ${o}`,
-          ],
+          ] : [`cp ${posix.join(i, props.llrtBinaryPath)} ${o}`],
```

Comment:
Not sure if this works well in some edge cases. Does it work when`forceDockerBundling: true` and/or `depsLockFilePath: 'some/child/packege-lock.json'`? I'll test those cases later.

Reply:
Also we may need an integration test to maintain this feature. I'll add one along the above test.

Reply:
Good spot, I'm not sure if it would work with `forceDockerBundling`. I imagine you would have to mount the directory somehow but I don't know if there's an option for that?

Reply:
@tmokmss I can take a look at the integration tests, does it just run with `yarn integ-runner`?

Reply:
yes. you may need to run yarn build for tsc though.

Reply:
Confirmed that as long as llrtBinaryPath is relative to [the `projectRoot` directory](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda_nodejs.NodejsFunction.html#projectroot), it works even with Docker. I added that information to jsdoc.